src/storage/AddVerticesProcessor.cpp

```diff
@@ -30,11 +32,16 @@ void AddVerticesProcessor::process(const cpp2::AddVerticesRequest& req) {
         std::for_each(vertices.begin(), vertices.end(), [&](auto& v){
             const auto& tags = v.get_tags();
             std::for_each(tags.begin(), tags.end(), [&](auto& tag) {
-                VLOG(4) << "PartitionID: " << partId << ", VertexID: " << v.get_id()
+                VLOG(3) << "PartitionID: " << partId << ", VertexID: " << v.get_id()
                         << ", TagID: " << tag.get_tag_id() << ", TagVersion: " << version;
                 auto key = NebulaKeyUtils::vertexKey(partId, v.get_id(),
                                                      tag.get_tag_id(), version);
                 data.emplace_back(std::move(key), std::move(tag.get_props()));
+                if (FLAGS_enable_vertex_cache && vertexCache_ != nullptr) {
+                    vertexCache_->evict(std::make_pair(v.get_id(), tag.get_tag_id()), partId);
```

Comment:
vertexCache_->insert?

Reply:
it looks as if...

Reply:
When updating vertex (insert/update/delete),  we just invalid the key in cache.

Reply:
Like this, Is it will be insert to the cache on the next hit?

Reply:
Yes.  The cache only be updated when reading.